### PR TITLE
Update main README file

### PR DESCRIPTION
**Pull request by adrianmcli**
n/t

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,70 @@
 # Drizzle Utils
 
-A library of frontend utilities for making dapps
+A library of tools for interacting with Ethereum smart contracts with a focus on [RxJS](https://github.com/ReactiveX/rxjs) streams so you can easily subscribe to events, contract state, and read-only methods.
```
Comment (honestbonsai) [inline README.md]: Perhaps we should talk about how Drizzle Utils fits into the greater hierarchy of things along with Drizzle and DRC. I don't thing it's clear to a new user which one to use.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,70 @@
...
+  artifact: contractArtifact
+})
+```
+
+### Streaming contract events
+
+Returns a Promise that resolves to an RxJS stream of events.
+
+```js
+const event$ = await drizzleUtils.createContractEvent$({
```
Comment (honestbonsai) [inline README.md]: This isn't a promise anymore

1. https://github.com/trufflesuite/drizzle-utils/blob/master/packages/core/index.js#L27
1. https://github.com/trufflesuite/drizzle-utils/blob/master/packages/contract-event-stream/index.js

File: README.md [hunk 3]
```diff
@@ -1,10 +1,70 @@
...
+
+event$.subscribe(event => console.log(event))
+```
+
+### Streaming the contract state
+
+Returns a Promise that resolves to an RxJS stream of contract states.
+
+```js
+const state$ = await drizzleUtils.createContractState$({
```
Comment (honestbonsai) [inline README.md]: This isn't a promise anymore

1. https://github.com/trufflesuite/drizzle-utils/blob/master/packages/core/index.js#L30
1. https://github.com/trufflesuite/drizzle-utils/blob/master/packages/contract-state-stream/index.js#L5

File: README.md [hunk 4]
```diff
@@ -1,10 +1,70 @@
...
+
+state$.subscribe(state => console.log(state))
+```
+
+### Streaming a constant read-only method from a contract
+
+In this example, `get` is a read-only method in our contract.
+
+```js
+const call$ = await drizzleUtils.createContractCall$({
```
Comment (honestbonsai) [inline README.md]: This isn't a promise anymore

1. https://github.com/trufflesuite/drizzle-utils/blob/master/packages/core/index.js#L24
1. https://github.com/trufflesuite/drizzle-utils/blob/master/packages/contract-call-stream/index.js#L4

File: README.md [hunk 5]
```diff
@@ -1,10 +1,70 @@
...
+})
+
+call$.subscribe(result => console.log(result))
+```
+
+### Using specific packages
+
+Refer to the specific package's README file for usage information.
 
 ## For development
```
Comment (honestbonsai) [inline README.md]: Is this section going to be filled out?

File: README.md (hunk 1 above)
Comment (adrianmcli) [reply README.md]: @honestbonsai I'm trying to shorten this as much as possible. I want this to be a one-sentence description, any ideas?

File: README.md (hunk 5 above)
Comment (adrianmcli) [reply README.md]: I thought this was sufficient, but if not maybe I'll clarify.

Comment (adrianmcli) [reply README.md]: @honestbonsai I've updated the text here, is it better?

File: README.md (hunk 1 above)
Comment (adrianmcli) [reply README.md]: @honestbonsai I've updated the description. Is it any better? Also, I think having "Drizzle" in the name is what might be confusing, but maybe we should just leave it for now. We can always rebrand after we get feedback from our first "soft release" to Truffle/ConsenSys.

File: README.md (hunk 5 above)
Comment (honestbonsai) [reply README.md]: @adrianmcli Sorry, I might not have seen the entire file, I was actually talking about `For development`, I dunno why but it looked empty to me earlier. But you do have something under there.

**Comment by honestbonsai** [comment]
The changes LGTM.

Build is failing cuz of timeout issues, but this is clearly not related to this PR. So merge when you want.

File: README.md (hunk 1 above)
Comment (honestbonsai) [reply README.md]: Yea it's probably the "Drizzle" part of the name. I'm fine with rebranding after the soft release.

**Comment by adrianmcli** [comment]
thx!

**Pull request merged**